WcaOnRails/db/structure.sql

```diff
@@ -1366,6 +1085,7 @@ CREATE TABLE `votes` (
 /*!50001 SET character_set_results     = utf8 */;
 /*!50001 SET collation_connection      = utf8_general_ci */;
 /*!50001 CREATE ALGORITHM=UNDEFINED */
+/*!50013 DEFINER=`root`@`localhost` SQL SECURITY DEFINER */
```

Comment:
I think I accidentally committed this change before and it caused problems. However, travis seems to be happy, so maybe this is okay to commit. Two options:

1. Paranoidly remove this from the diff. Bonus points for figuring out how this happened.
2. Be fearless, merge this up, and keep an eye out for any problems that may arise from this (probably related to developers seeding their databases).